### DISCUSSION TITLE
Allow to obtain the count and a list of updated packages in grouped dependencies

### DISCUSSION THREAD

**Discussion by phgogo** [renovatebot/renovate#20645] [Suggest an Idea]
### What would you like Renovate to be able to do?

I would like to be able to do something like this, but currently there are no template fields available to support this.

```
{
      "groupName": "tf_module_patch",
      "commitMessagePrefix": "{{updateType}}: [Renovate] New Terraform Module Updates: {{#if isGroup}}Multiple Packages{{else}}{{depName}}{{/if}}"
}
```

Or even better:

```
{
      "groupName": "tf_module_patch",
      "commitMessagePrefix": "{{updateType}}: [Renovate] New Terraform Module Updates: {{ commaSeparatedListOfDeps }}"
}
```

### If you have any ideas on how this should be implemented, please tell us here.

- Add template fields for:
  - Count of updated packages
  - Comma separated list of updated packages
  - Boolean if there is more than one updated Project, e.g. `isGroup`

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by rarkins** [comment]
Hi, I have converted this from an Issue to a Discussion.

You can already customize behavior of grouped dependencies using the `group` object.

e.g.

```
{
  "group": {
	"commitMessagePrefix": "..."
  }
}
```

If this doesn't satisfy, please create a minimal reproduction repo to demonstrate
